lib/content-services/src/lib/document-list/components/file-auto-download/file-auto-download.component.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component } from '@angular/core';
+import { FileAutoDownloadActionsEnum } from '../../models/file-auto-download-actions.enum';
+
+@Component({
+  selector: 'adf-file-auto-download',
+  templateUrl: './file-auto-download.component.html',
+})
+export class FileAutoDownloadComponent {
+    FileAutoDownloadActionsEnum = FileAutoDownloadActionsEnum;
+    constructor() {}
```

Comment:
you don't need empty constructor, this should be raising linting warnings I think

Reply:
Removed empty constructor